### DISCUSSION TITLE
Add option to display Capital letters in on-screen keymap

### DISCUSSION THREAD

**Discussion by precompute** [monkeytypegame/monkeytype#1174] [Ideas] [answered]
Helps improve legibility with some fonts (custom or otherwise)

**Comment by frogfrogfrogfrogfrog** [comment]
I think this is a nice idea. Since many fonts make I and L look extremely similar.

**Comment by Miodec** [comment, marked as answer]
Added #1183

**Reply by precompute** [reply]
Has been implemented!  Thaks a ton!
